cluster/juju/layers/kubernetes-worker/actions/restart

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+set -u
+set -x
+
+# Initiate a service restart without the requirement of rebooting the unit.
+# Optional parameters:
+# force: - Do not pause/cordon the unit during service restart
+# docker: - Cycle the docker daemon during the kubernetes-worker service reboot
+
+FORCE=$(action-get force)
+DOCKER=$(action-get docker)
+
+# Force restarts are kind of a bonzai thing. We're not concerned with the
+# Workloads contained therein, so forcibly recycle the services and exit 0.
+
+if [ ${FORCE} == "True" ]; then
+
+  if [ ${DOCKER} == "True" ]; then
+    systemctl "Force restarting docker service..."
```

Comment:
Oops, missed this before. @lazyPower I hit an error here, should be `juju-log` not `systemctl` yeah?